docs-chef-io/content/server/server_services.md

```diff
@@ -80,7 +80,12 @@ The following services are part of the Chef Infra Server:
 - nginx
 - opscode-erchef
 - postgresql
-- redis-lb
+- redis_lb 
+
+Note: 
```

Comment:
**Replace it with:** {{< note >}} From Chef Infra Server 15.10. x onwards, the [KeyDB] (https://docs.keydb.dev/docs/about) service replaces Redis. The configuration name and service name will continue to be `redis_lb`. {{< note/ >}}